app/src/main/java/com/example/hris/repository/HrisRepository.kt

```diff
@@ -11,15 +13,86 @@ import javax.inject.Singleton
 @Singleton
 class HrisRepository @Inject constructor(
     private val hrisDao: HrisDao
-){
+) {
 
-    val profileData: LiveData<User> = hrisDao.getProfile()
+    val loginData = MutableLiveData<User>()
```

Comment:
The `loginData` and the `userData` fields virtually return the same data. Consider the concept of SSOT — Single Source of Truth. There should only be 1 source for the same data.